static/app/views/performance/utils/useModuleURL.tsx

```diff
@@ -31,15 +31,31 @@ export const MODULE_BASE_URLS: Record<ModuleName, string> = {
 type ModuleNameStrings = `${ModuleName}`;
 type RoutableModuleNames = Exclude<ModuleNameStrings, '' | 'other'>;
 
-export const useModuleURL = (moduleName: RoutableModuleNames): string => {
+export const useModuleURL = (
+  moduleName: RoutableModuleNames,
+  bare: boolean = false
+): string => {
+  const builder = useModuleURLBuilder();
+  return builder(moduleName, bare);
+};
+
+type URLBuilder = (moduleName: RoutableModuleNames, bare?: boolean) => string;
+
+export function useModuleURLBuilder(): URLBuilder {
   const {slug} = useOrganization();
 
-  if (moduleName === ModuleName.AI) {
-    // AI Doesn't live under "/performance"
-    return normalizeUrl(`/organizations/${slug}/${AI_BASE_URL}`);
-  }
+  return function (moduleName: RoutableModuleNames, bare: boolean = false) {
```

Comment:
Why is `bare = false`, I feel like `true` makes more sense as a default.

Also, I think `useModuleURLBuilder` should take in `bare`, so then the `builder` just inherits those settings and you can enforce a common set of module routes have the same path.

Reply:
`bare` is `false` by default because most usage of `useModuleURL` expect it to return the full URL, the sidebar is the exception. I can set it to `true` by default though, I don't have a strong opinion.

As for `useModuleURLBuilder` taking in `bare` as a parameter, that works for me, I can make the change

Reply:
> the sidebar is the exception

Yes you're totally right, I just forgot about that, let's keep it as `false` then!

Reply:
Done!